.babelrc-v9.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 204 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27036&commitId=97655de6abbb2b30a7f098815dfa1745f8012058&env=prod)

✅ There was 0 screenshots added, 125 screenshots removed, 843 screenshots unchanged, 0 screenshots with different dimensions and 79 screenshots with visible difference.


  <details><summary><b>OverflowSet.Root - RTL.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root - RTL.default.chromium_1.png|0| Removed|

</details>


  <details><summary><b>OverflowSet.Root.default.chromium</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root.default.chromium_1.png|0| Removed|

</details>


  <details><summary><b>unknown</b> 202 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Beak 25.default.chromium.png|1607213| Changed|
|Callout.Bottom auto edge.default.chromium.png|1607217| Changed|
|Callout.Bottom center.default.chromium.png|1607217| Changed|
|Callout.Bottom left edge - RTL.default.chromium.png|1607217| Changed|
|Callout.Bottom left edge.default.chromium.png|1607217| Changed|
|Callout.Bottom right edge - RTL.default.chromium.png|1607217| Changed|
|Callout.Bottom right edge.default.chromium.png|1607217| Changed|
|Callout.Gap space 25.default.chromium.png|1607217| Changed|
|Callout.Left bottom edge.default.chromium.png|1607222| Changed|
|Callout.Left center.default.chromium.png|1607222| Changed|
|Callout.Left top edge.default.chromium.png|1607221| Changed|
|Callout.No beak.default.chromium.png|1607222| Changed|
|Callout.No callout width specified.default.chromium.png|1607123| Changed|
|Callout.Rendering callout attached to a rectangle.default.chromium.png|1604310| Changed|
|Callout.Right bottom edge.default.chromium.png|1607210| Changed|
|Callout.Right center.default.chromium.png|1607210| Changed|
|Callout.Right top edge.default.chromium.png|1607209| Changed|
|Callout.Root.default.chromium.png|1607217| Changed|
|Callout.Top auto edge.default.chromium.png|1607199| Changed|
|Callout.Top center.default.chromium.png|1607198| Changed|
|Callout.Top left edge.default.chromium.png|1607199| Changed|
|Callout.Top right edge.default.chromium.png|1607198| Changed|
|Coachmark.Collapsed.default.chromium.png|1609091| Changed|
|Coachmark.Rendering Coachmark attached to a rectangle.default.chromium.png|1606329| Changed|
|ContextualMenu.Root.default.chromium.png|1609190| Changed|
|ContextualMenu.With headers.default.chromium.png|1609283| Changed|
|ContextualMenu.With icons.default.chromium.png|1609248| Changed|
|ContextualMenu.With secondaryText - RTL.default.chromium.png|1608426| Changed|
|ContextualMenu.With secondaryText.default.chromium.png|1608585| Changed|
|ContextualMenu.With split button submenu.default.chromium.png|1609705| Changed|
|ContextualMenu.With submenu - RTL.default.chromium.png|1609622| Changed|
|ContextualMenu.With submenu.default.chromium.png|1609634| Changed|
|ContextualMenu.With submenus with hrefs.menu opened.chromium.png|1608515| Changed|
|Dialog.Blocking.default.chromium.png|279107| Changed|
|Dialog.Large header.default.chromium.png|279786| Changed|
|Dialog.Root - RTL.default.chromium.png|286741| Changed|
|Dialog.Root.default.chromium.png|279133| Changed|
|Dialog.Wide Dialog.default.chromium.png|307996| Changed|
|FocusTrapZones.Dialog nested in Panel - RTL.default.chromium.png|905416| Changed|
|FocusTrapZones.Dialog nested in Panel.default.chromium.png|905299| Changed|
|FocusTrapZones.Panel on its own.default.chromium.png|1606637| Changed|
|HoverCard.Root - RTL.fully expanded with test content.chromium.png|1607580| Changed|
|HoverCard.Root.fully expanded with test content.chromium.png|1607590| Changed|
|Image.Maximize frame, portrait container.default.chromium.png|233773| Changed|
|Keytip.Disabled.default.chromium.png|235821| Changed|
|Keytip.Offset.default.chromium.png|235696| Changed|
|Keytip.Root.default.chromium.png|235743| Changed|
|Layer.Root - RTL.default.chromium.png|1609749| Changed|
|Layer.Root.default.chromium.png|1609750| Changed|
|Modal.Modeless.default.chromium.png|1606601| Changed|
|Modal.Root.default.chromium.png|897858| Changed|
|Nav.Disabled - RTL.default.chromium.png|284643| Changed|
|Nav.Disabled.default.chromium.png|277557| Changed|
|Nav.Root - RTL.default.chromium.png|306432| Changed|
|Nav.Root.default.chromium.png|298775| Changed|
|Panel.Custom anchored left.default.chromium.png|1606889| Changed|
|Panel.Custom.default.chromium.png|1609495| Changed|
|Panel.Extra large right.default.chromium.png|1606890| Changed|
|Panel.Large fixed right.default.chromium.png|1606770| Changed|
|Panel.Large right.default.chromium.png|1606992| Changed|
|Panel.Medium right - RTL.default.chromium.png|1606849| Changed|
|Panel.Medium right.default.chromium.png|1606850| Changed|
|Panel.SearchBox and Right Panel - RTL.default.chromium.png|1609497| Changed|
|Panel.SearchBox and Right Panel.default.chromium.png|1609497| Changed|
|Panel.Small fixed right w- close button - RTL.default.chromium.png|1606766| Changed|
|Panel.Small fixed right w- close button.default.chromium.png|1606766| Changed|
|Panel.Small fluid right.default.chromium.png|1609361| Changed|
|Panel.Small left w- close button - RTL.default.chromium.png|1607029| Changed|
|Panel.Small left w- close button.default.chromium.png|1607080| Changed|
|Panel.With custom navigation.default.chromium.png|1606346| Changed|
|Panel.With footer at the bottom.default.chromium.png|1604847| Changed|
|Panel.With no header, close button.default.chromium.png|1607168| Changed|
|Panel.With no navigation.default.chromium.png|1606689| Changed|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|1603225| Changed|
|TeachingBubble.SmallHeadline.default.chromium.png|243666| Changed|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|271484| Changed|
|TeachingBubble.WideIllustration.default.chromium.png|263091| Changed|
|Tooltip - Multiple.Two Tooltips.hover outer.chromium.png|1608863| Changed|
|Tooltip.Default.default.chromium.png|1609457| Changed|
|Breadcrumb.Button - RTL.click.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button.click.chromium.png|0| Removed|
|Breadcrumb.Button.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.actionable overflow hover.chromium.png|0| Removed|
|Breadcrumb.Hovering items.non-actionable overflow hover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.click.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root.click.chromium.png|0| Removed|
|Breadcrumb.Root.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Root.shortTitleHover.chromium.png|0| Removed|
|Button Command (compat).Checked.open.chromium.png|0| Removed|
|Button Command (compat).Root - RTL.open.chromium.png|0| Removed|
|Button Command (compat).Root.open.chromium.png|0| Removed|
|ComboBox.Error.click.chromium.png|0| Removed|
|ComboBox.Placeholder.click.chromium.png|0| Removed|
|ComboBox.Root - RTL.click.chromium.png|0| Removed|
|ComboBox.Root.click.chromium.png|0| Removed|
|ComboBox.Styled.click.chromium.png|0| Removed|
|CommandBar.Icons only.click.chromium.png|0| Removed|
|CommandBar.Root - RTL.click.chromium.png|0| Removed|
|CommandBar.Root.click.chromium.png|0| Removed|
|CommandBar.Text only.click.chromium.png|0| Removed|
|ContextualMenu.Root.click.chromium.png|0| Removed|
|ContextualMenu.Root.hover.chromium.png|0| Removed|
|ContextualMenu.With headers.click.chromium.png|0| Removed|
|ContextualMenu.With headers.hover.chromium.png|0| Removed|
|ContextualMenu.With icons.click.chromium.png|0| Removed|
|ContextualMenu.With icons.hover.chromium.png|0| Removed|
|ContextualMenu.With secondaryText - RTL.click.chromium.png|0| Removed|
|ContextualMenu.With secondaryText - RTL.hover.chromium.png|0| Removed|
|ContextualMenu.With secondaryText.click.chromium.png|0| Removed|
|ContextualMenu.With secondaryText.hover.chromium.png|0| Removed|
|ContextualMenu.With split button submenu.click.chromium.png|0| Removed|
|ContextualMenu.With split button submenu.hover.chromium.png|0| Removed|
|ContextualMenu.With submenu - RTL.click.chromium.png|0| Removed|
|ContextualMenu.With submenu - RTL.hover.chromium.png|0| Removed|
|ContextualMenu.With submenu.click.chromium.png|0| Removed|
|ContextualMenu.With submenu.hover.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.item1 hovered.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.item2 hovered.chromium.png|0| Removed|
|ContextualMenu.With submenus with hrefs.parent hovered.chromium.png|0| Removed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.click.chromium.png|0| Removed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover day.chromium.png|0| Removed|
|DatePicker.Allow text input.click.chromium.png|0| Removed|
|DatePicker.Allow text input.hover day.chromium.png|0| Removed|
|DatePicker.Allow text input.hover month.chromium.png|0| Removed|
|DatePicker.Placeholder.click.chromium.png|0| Removed|
|DatePicker.Placeholder.hover day.chromium.png|0| Removed|
|DatePicker.Placeholder.hover month.chromium.png|0| Removed|
|DatePicker.Required.click.chromium.png|0| Removed|
|DatePicker.Required.hover day.chromium.png|0| Removed|
|DatePicker.Required.hover month.chromium.png|0| Removed|
|DatePicker.Root - RTL.click.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover day.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover month.chromium.png|0| Removed|
|DatePicker.Root.click.chromium.png|0| Removed|
|DatePicker.Root.hover day.chromium.png|0| Removed|
|DatePicker.Root.hover month.chromium.png|0| Removed|
|DatePicker.Underlined and Required.click.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover day.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover month.chromium.png|0| Removed|
|DatePicker.Underlined.click.chromium.png|0| Removed|
|DatePicker.Underlined.hover day.chromium.png|0| Removed|
|DatePicker.Underlined.hover month.chromium.png|0| Removed|
|Dropdown.Custom Dropdown - RTL.click.chromium.png|0| Removed|
|Dropdown.Custom Dropdown - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.click.chromium.png|0| Removed|
|Dropdown.Custom Dropdown.hover item.chromium.png|0| Removed|
|Dropdown.Disabled option selected.click.chromium.png|0| Removed|
|Dropdown.Disabled option selected.hover item.chromium.png|0| Removed|
|Dropdown.Multiselect - RTL.click.chromium.png|0| Removed|
|Dropdown.Multiselect - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Multiselect.click.chromium.png|0| Removed|
|Dropdown.Multiselect.hover item.chromium.png|0| Removed|
|Dropdown.Required without label.click.chromium.png|0| Removed|
|Dropdown.Required without label.hover item.chromium.png|0| Removed|
|Dropdown.Required.click.chromium.png|0| Removed|
|Dropdown.Required.hover item.chromium.png|0| Removed|
|Dropdown.Root - RTL.click.chromium.png|0| Removed|
|Dropdown.Root - RTL.hover item.chromium.png|0| Removed|
|Dropdown.Root.click.chromium.png|0| Removed|
|Dropdown.Root.hover item.chromium.png|0| Removed|
|FocusTrapZones.Dialog nested in Panel - RTL.click on panel close button.chromium.png|0| Removed|
|FocusTrapZones.Dialog nested in Panel.click on panel close button.chromium.png|0| Removed|
|FocusTrapZones.Panel on its own.click on panel close button.chromium.png|0| Removed|
|Nav.Disabled - RTL.click.chromium.png|0| Removed|
|Nav.Disabled - RTL.hover.chromium.png|0| Removed|
|Nav.Disabled.click.chromium.png|0| Removed|
|Nav.Disabled.hover.chromium.png|0| Removed|
|Nav.Root - RTL.click.chromium.png|0| Removed|
|Nav.Root - RTL.hover.chromium.png|0| Removed|
|Nav.Root.click.chromium.png|0| Removed|
|Nav.Root.hover.chromium.png|0| Removed|
|Panel.SearchBox and Right Panel - RTL.click.chromium.png|0| Removed|
|Panel.SearchBox and Right Panel.click.chromium.png|0| Removed|
|PeoplePicker.Compact selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Compact selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Compact.suggestions.chromium.png|0| Removed|
|PeoplePicker.List selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.List selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.List.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal selected.suggestions.chromium.png|0| Removed|
|PeoplePicker.Normal.suggestions.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|
|ResizeGroup.Root - RTL.click overflow.chromium.png|0| Removed|
|ResizeGroup.Root.click overflow.chromium.png|0| Removed|
|TagPicker.Root.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Root.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Selected - RTL.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Selected - RTL.Suggestion Menu Item Hover.chromium.png|0| Removed|
|TagPicker.Selected.Open Suggestion Menu.chromium.png|0| Removed|
|TagPicker.Selected.Suggestion Menu Item Hover.chromium.png|0| Removed|
|Tooltip - Multiple.Two Tooltips.hover inner.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>